src/__tests__/translate-test.js

```diff
@@ -1,7 +1,7 @@
 const translate = require("../translate").default
 const TranslatorProvider = require("../TranslatorProvider").default
 const React = require("react")
-const { renderIntoDocument } = require("react-addons-test-utils")
+const renderIntoDocument = require("react-dom/lib/ReactTestUtils").renderIntoDocument
```

Comment:
Yep, you're right. I was having issues with tests because I was an idiot and forgot to update `yarn.lock`.